pages/index.js

```diff
@@ -1,7 +1,66 @@
-export default function HomePage() {
+import { shoppingItems } from "../lib/shoppingItemsData.js";
+import { categories } from "../lib/categoriesData.js";
+
+import styled from "styled-components";
+
+export default function ShoppingItemsList() {
+  const categoryColorShoppingItems = shoppingItems.map((shoppingItem) => {
```

Comment:
```suggestion
  const shoppingItemsWithColor = shoppingItems.map((shoppingItem) => {
```
"categoryColorShoppingItems" is a "weird" name for the list you are creating. You are returning the shoppingList and just add the color, so it would make more sense to give a more clear name.

Reply:
Hey Stephan, thanks for your comments. I had parsed categoryColor as a modifier, but your comment makes perfect sense. I've changed it to "shoppingItemsWithCategoryColor".